access/ms-teams/configure.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"archive/zip"
+	"embed"
+	"fmt"
+	"html/template"
+	"io"
+	"os"
+	"path"
+
+	"github.com/google/uuid"
+	"github.com/gravitational/teleport-plugins/lib"
+	"github.com/gravitational/trace"
+)
+
+const (
+	// TODO: Replace with live URL
+	guideURL = "http://example.com"
```

Comment:
:nail_care: 
We can use `https://goteleport.com/docs/enterprise/workflow/` for now, but let's keep the TODO